src/components/selections/attributes.js

```diff
@@ -54,6 +54,51 @@ module.exports = overrideAll(templatedArray('selection', {
         description: 'Sets the selection\'s end y position.'
     },
 
+    x0shift: {
+        valType: 'number',
+        dflt: 0,
+        min: -1,
```

Comment:
For `selections` limiting the range to be between -0.5 and 0.5 is safer IMHO.

Reply:
As discussed, I removed shift attributes from selections for now.